.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,48 @@
+---
+name: "\U0001F680Feature request"
+about: Suggest a feature for rules_python
+
+---
+<!--🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅
+
+Oh hi there! 😄 
+
+To expedite issue processing please search open and closed issues before submitting a new one.
+Existing issues often contain information about workarounds, resolution, or progress updates.
+
+🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅-->
+
+
+# 🚀 feature request
+
+### Relevant Rules
+
+<!-- Tell us if you want to add a feature to an existing rule or add a new rule -->
+
+<!--
+
+If you're looking for a new rule first make sure you check awesome-bazel for an
+curated list of existing bazel rules https://github.com/jin/awesome-bazel
+
+If you don't find what you're looking for there, before opening a feature request make sure
+this repository is the right place for that.
+
+We have a list of requirements that the rules in this repository need to follow.
+You can check them in the README.md file of the project.
```

Comment:
nit: this is a bit vague about which "requirements" are being referred to in the README. Looking at the README now I can't pick out the relevant section.

Reply:
oops, good catch, that's copy-pasta from rules_nodejs